src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -73,12 +72,17 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    void ColorSchemes::Edit_Click(const IInspectable& /*sender*/, const RoutedEventArgs& /*e*/)
+    {
+        _ViewModel.RequestEditSelectedScheme();
```

Comment:
confused - why isn't this hooked up directly to the viewmodel from the xaml? no need for an event handler in the View

Reply:
yup, that's exactly what I missed haha. Didn't realize you can bind a click handler directly to the view model :P